### ISSUE TITLE
Log CultureInfo on  Host startup

### ISSUE THREAD

**Issue by grahamehorner** [Azure/azure-functions-host#3385]
The AzureFunctions host should be able to run under a different culture to that for the host operating system and by default run under the en-US culture so to avoid any culture related development issues.

It would also be good to display the configured/default culture at runtime for local debugging.

**Comment by pragnagopa** [comment]
@grahamehorner - Do you see any issues when not using en-US culture?

**Comment by grahamehorner** [comment]
Yes, we have a number of functions that perform conversion of data and not begin aware of the culture in logging can be confusing when dates and currency are concerned.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
